python/examples/integration/integration_client.py

```diff
@@ -0,0 +1,67 @@
+from __future__ import print_function
+
+import time
+import argparse
+
+import grpc
+from jaeger_client import Config
```

Comment:
doesn't some sort of dependency need to be added to setup.py or similar for this to work?